region/lowernorfair/west.json

```diff
@@ -984,22 +995,21 @@
                   "requires": [
                     "h_canNavigateHeatRooms",
                     "canTrickyJump",
+                    "canPreciseWalljump",
```

Comment:
I think I would rather remove this whole strat. The timing is basically the same as the crumble jump, but also requires a very precise jump. 

With crumble jump off, it would be weird to still be required to do this. I could see the only options when entering the room to be to get the item and then go down to GT